### ISSUE TITLE
ImportFullyQualifiedNamesRector results in weird change 

### ISSUE THREAD

**Issue by matiit** [rectorphp/rector#1527]
Command that I used:

`docker run -v $(pwd):/project rector/rector:latest process /project/src --autoload-file /project/vendor/autoload.php --level symfony40`

Refactor performed:

```
@@ -67,7 +67,7 @@ class Stage
      * Stage constructor.
      *
      * @param string $stage
-     * @param boolean $undefined Should the object be undefined?
+     * @param boolean $undefined Should the object be undefined??d??d??d??
      */
```

which does look pretty weird. 
The class that has been modified is a pretty simple PHP5.6 class (compatible with PHP7, but not using PHP7 features.

If more info is needed, plesase let me know.

Thank you.

== 

Edit,
I can see that the 'rector' that has been applied is:
`Rector\CodingStyle\Rector\Namespace_\ImportFullyQualifiedNamesRector`

**Comment by Aerendir** [comment]
Are you able to reproduce the issue with a failing test case?

If you need a guidance about how to create one, you can check the opened PR that contain already submitted failing tests.

**Comment by TomasVotruba** [comment]
Fixed by #1549   

Thanks for reporting :+1:
